__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -1744,10 +1766,13 @@ describe('getTxHistory', () => {
     // HTR transaction incoming
     const tx1 = await GenesisWalletHelper.injectFunds(hWallet.getAddressAtIndex(0), 10);
     txHistory = await hWallet.getTxHistory();
-    expect(txHistory).toHaveLength(1);
-    expect(txHistory[0].txId).toEqual(tx1.hash);
-    expect(txHistory[0].tokenUid).toEqual(HATHOR_TOKEN_CONFIG.uid);
-    expect(txHistory[0].balance).toEqual(10);
+    expect(txHistory).toStrictEqual([
+      expect.objectContaining({
+        txId: expect.any(String),
```

Comment:
Why not `tx1.hash`, as before?

Reply:
Fixed it on [dc82fee](https://github.com/HathorNetwork/hathor-wallet-lib/pull/393/commits/dc82fee1edfeb1867d4e95902343c36393456b22), it was supposed to stay as `tx1.hash`